packages/react/test/specs/components/Accordion/AccordionTitle-test.tsx

```diff
@@ -0,0 +1,143 @@
+  isConformant(AccordionTitle, {
+    eventTargets: {
+      onClick: `.${AccordionTitle.slotClassNames.button}`,
+    },
+    requiredProps: {
+      buttonRef: _.noop,
+    },
+  })
+
+  describe('accessiblity', () => {
+    describe('header', () => {
+      handlesAccessibility(AccordionTitle, {
+        requiredProps: {
+          as: 'h3',
+          buttonRef: _.noop,
+        },
+        defaultRootRole: undefined,
+      })
+    })
+    describe('div header', () => {
+      handlesAccessibility(AccordionTitle, {
+        requiredProps: {
+          buttonRef: _.noop,
+        },
+        defaultRootRole: 'heading',
+      })
+    })
+
+    describe('aria-disabled', () => {
+      test('is set to true, if active and cannot be collapsed', () => {
+        const renderedComponent = mountWithProviderAndGetComponent(
+          AccordionTitle,
+          <AccordionTitle active canBeCollapsed={false} buttonRef={_.noop} />,
+        )
+        expect(
+          getRenderedAttribute(
+            renderedComponent,
+            'aria-disabled',
+            `.${AccordionTitle.slotClassNames.button}`,
+          ),
+        ).toBe('true')
+      })
+
+      test('is set to false, if active and can be collapsed', () => {
+        const renderedComponent = mountWithProviderAndGetComponent(
+          AccordionTitle,
+          <AccordionTitle active canBeCollapsed buttonRef={_.noop} />,
+        )
+        expect(
+          getRenderedAttribute(
+            renderedComponent,
+            'aria-disabled',
+            `.${AccordionTitle.slotClassNames.button}`,
+          ),
+        ).toBe('false')
+      })
+
+      test('is set to false, if not active', () => {
+        const renderedComponent = mountWithProviderAndGetComponent(
+          AccordionTitle,
+          <AccordionTitle buttonRef={_.noop} />,
+        )
+        expect(
+          getRenderedAttribute(
+            renderedComponent,
+            'aria-disabled',
+            `.${AccordionTitle.slotClassNames.button}`,
+          ),
+        ).toBe('false')
+      })
+    })
+
+    describe('aria-expanded', () => {
+      test('is set to true, if actived', () => {
+        const renderedComponent = mountWithProviderAndGetComponent(
+          AccordionTitle,
+          <AccordionTitle buttonRef={_.noop} active />,
+        )
+        expect(
+          getRenderedAttribute(
+            renderedComponent,
+            'aria-expanded',
+            `.${AccordionTitle.slotClassNames.button}`,
+          ),
+        ).toBe('true')
+      })
+
+      test('is set to true, if actived', () => {
+        const renderedComponent = mountWithProviderAndGetComponent(
+          AccordionTitle,
+          <AccordionTitle buttonRef={_.noop} />,
+        )
+        expect(
+          getRenderedAttribute(
+            renderedComponent,
+            'aria-expanded',
+            `.${AccordionTitle.slotClassNames.button}`,
+          ),
+        ).toBe('false')
```

Comment:
Test description is wrong here. Again, let's move all behavior tests from the component's test.